roofit/roofitcore/src/RooAbsMinimizerFcn.cxx

```diff
@@ -0,0 +1,526 @@
+/*****************************************************************************
+ * Project: RooFit                                                           *
+ * Package: RooFitCore                                                       *
+ * @(#)root/roofitcore:$Id$
+ * Authors:                                                                  *
+ *   AL, Alfio Lazzaro,   INFN Milan,        alfio.lazzaro@mi.infn.it        *
+ *   PB, Patrick Bos, Netherlands eScience Center, p.bos@esciencecenter.nl   *
+ *                                                                           *
+ *                                                                           *
+ * Redistribution and use in source and binary forms,                        *
+ * with or without modification, are permitted according to the terms        *
+ * listed in LICENSE (http://roofit.sourceforge.net/license.txt)             *
+ *****************************************************************************/
+
+#ifndef __ROOFIT_NOROOMINIMIZER
+
+//////////////////////////////////////////////////////////////////////////////
+//
+// RooAbsMinimizerFcn is an interface class to the ROOT::Math function
```

Comment:
Make this readable for doxygen, please:
- Either `///` everywhere
- Or `/** ... * ... */
- And
```c++
/// \class RooAbsMinimizerFcn
/// Once-sentence description.
///
/// More description.
```

Reply:
For what's following I don't have enough time. I assume this is covered by tests?